examples/nas/cream/Cream.md

```diff
@@ -0,0 +1,94 @@
+# Cream of the Crop: Distilling Prioritized Paths For One-Shot Neural Architecture Search

```

Comment:
Please put a link to documentation here instead of copying the full file. Refer to other examples.

Reply:
Updated. I will add a Arxiv link in the near future